src/main/kotlin/xyz/atrius/waystones/data/KeyState.kt

```diff
@@ -1,14 +1,15 @@
 package xyz.atrius.waystones.data
 
 import org.bukkit.Location
+import xyz.atrius.waystones.localization
 
 sealed class KeyState(val message: String?) {
 
     object None : KeyState(null)
 
-    object Severed : KeyState("The link to this waystone has been severed")
+    object Severed : KeyState(localization.localize("key-severed"))
 
-    object Blocked : KeyState("You are too sick to warp")
+    object Blocked : KeyState(localization.localize("key-blocked"))
```

Comment:
If needed, please avoid marking these as `var`. It would likely make more sense to convert these to a high-order function. Example:

```kt
sealed class KeyState(val message: () -> String?)

// ...

/*
 * If the last argument is a high-order function you should be able to place it outside 
 * of the (), and omit them entirely if the function is the only argument passed. I'm not
 * completely sure how this behaves with constructors specifically though.
 */
object Severed : KeyState { localization.localize("key-severed") }
```

Reply:
I was thinking about returning a `LocalizedString` instead, which overrides invoke() and toString() to return `localization.localize` result. It may be better than returing function references.

Reply:
I'm not sure honestly, you'd need to still call toString() implicitly in order for it to function as expected anyways. Depending how you write it, it could be a fairly hacky solution too.

Reply:
I have added LocalizedString class, which seems to work fine.